crates/sui-indexer-alt-graphql/src/api/types/gas_input.rs

```diff
@@ -37,7 +40,11 @@ type CGasPayment = JsonCursor<usize>;
 impl GasInput {
     /// Address of the owner of the gas object(s) used.
     async fn gas_sponsor(&self) -> Option<Address> {
-        Some(Address::with_address(self.scope.clone(), self.native.owner))
+        if self.native.owner == SuiAddress::ZERO {
+            None
+        } else {
+            Some(Address::with_address(self.scope.clone(), self.native.owner))
```

Comment:
nit: this is the pattern we use elsewhere when we do the same thing:

```suggestion
        (self.native.owner != SuiAddress::ZERO).then(|| {
            Address::with_address(self.scope.clone(), self.native.owner)
        })
```